server/repository/src/migrations/v2_06_00/add_masterlist_id_to_invoice.rs

```diff
@@ -0,0 +1,21 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "add_masterlist_id_to_invoice"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        sql!(
+            connection,
+            r#"
+                ALTER TABLE invoice ADD COLUMN master_list_id TEXT
```

Comment:
Since this is linking the invoice to program, should be renamed to program_id and reference the program table. Could you also do a migration to add and save the period_id as well? Might need mSupply implementation for period_id too for syncing